e2e/fixtures/respondent.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  name: 'Joe Bloggs',
+  dob: {
+    day: '01',
+    month: '01',
+    year: '1980',
+  },
+  gender: 'male',
```

Comment:
this should be capital M if youre passing it into the dropdown field, `Male`

Reply:
just refactored to use the same GenderList as FPL-39, making the changes in the tests now 👍

Reply:
good boy

Reply:
*good male

Reply:
male or Male doesn't appear to make a difference

Reply:
It doesn't or my tests would fail and say the option doesn't exist. Changed to M anyway.